action-allowedlist/approved.json

```diff
@@ -163,18 +163,34 @@
     {
         "actionLink": "github/codeql-action/init",
         "actionVersion": "1fae5bf71b0ecdc7d0a2ef0d0c28409d99693966",
-        "tag": "2.9.2"
+        "tag": "codeql-bundle-20220512"
```

Comment:
CodeQL's tags are like "codeql-bundle-<date>". You have to click into them to find the version number for the codeql CLI (example: 2.9.2). The actual tag on the release is much more useful.